Makefile

```
@@ -176,6 +176,44 @@ build-mcuboot: submodules zephyr fprime-venv
 
+##@ Debugging / OpenOCD
+
+OPENOCD_DIR ?= $(shell pwd)/tools/openocd
+OPENOCD_REPO ?= https://github.com/raspberrypi/openocd.git
+OPENOCD_REF ?= acff23f
+OPENOCD_BIN ?= $(OPENOCD_DIR)/src/openocd
+OPENOCD_JOBS ?= 4
+OPENOCD_FLASH_SPEED ?= 5000
+OPENOCD_COMMON_FLAGS ?= -s $(OPENOCD_DIR)/tcl -f interface/cmsis-dap.cfg -f target/rp2350.cfg -c "adapter speed $(OPENOCD_FLASH_SPEED)"
+
+$(OPENOCD_DIR)/.built:
```

checkmake (0.3.2)

[warning] Target body for "$(OPENOCD_DIR)/.built" exceeds allowed length of 5 lines (9).

(maxbodylength)